src/components/WeeklyReport/ReportActivity.jsx

```diff
@@ -1,24 +1,45 @@
-import React, { Component } from 'react';
-import { Row } from 'react-bootstrap';
-import DateUtilities from '../../utilities/DateUtilities';
-import './ReportActivity.css';
+import { Component } from "react";
+import { Row } from "react-bootstrap";
+import DateUtilities from "../../utilities/DateUtilities";
+import "./ReportActivity.css";
 
 class ReportActivity extends Component {
-
-    render() {
-        let activity = this.props.activity;
-        return (
-            <Row className="activity">
-                <p className="preserve-whitespace">
-                    {this.props.hideWeekOf ? '' : <><strong>Week of:</strong> {DateUtilities.getDate(activity.WeekOf).format("MM/DD/YYYY")} <br /></>}
-                    <strong>Activity/Purpose:</strong> {activity.Title}<br />
-                    <strong>Action Taken/In Work:</strong> {activity.ActionTaken}<br />
-                    <strong>OPRs:</strong> {activity.OPRs && activity.OPRs.results && activity.OPRs.results.map((OPR, index, array) =>
-                        (<span key={OPR.Id}> {OPR.Title}{array.length - 1 > index ? ';' : ''} </span>))}
-                </p>
-            </Row>
-        );
-    }
+  render() {
+    let activity = this.props.activity;
+    const MAR = this.props.MAR;
+    return (
+      <Row className="activity">
+        <p className="preserve-whitespace">
+          {this.props.hideWeekOf ? (
+            ""
+          ) : (
+            <>
+              <strong>Week of:</strong>{" "}
+              {DateUtilities.getDate(activity.WeekOf).format("MM/DD/YYYY")}{" "}
+              <br />
+            </>
+          )}
+          <strong>Activity/Purpose:</strong> {activity.Title}
+          <br />
+          <strong>Action Taken/In Work:</strong>{" "}
+          {MAR
```

Comment:
Like that you added a comment for this, would it be better written (clearer) as "MAR && activity.MARText ? activity.MARText : activity.ActionTaken"